### PR TITLE
[ticket/14192] Add core.memberlist_memberrow_before event

### PR DESCRIPTION
**Pull request by Senky**
[PHPBB3-14192](https://tracker.phpbb.com/browse/PHPBB3-14192)

### CODE REVIEW

File: phpBB/memberlist.php [hunk 1]
```diff
@@ -1448,6 +1448,20 @@
 				usort($user_list,  'phpbb_sort_last_active');
 			}
 
+			// do we need to display contact fields as such
+			$use_contact_fields = false;
+
+			/**
+			 * Modify the query used to get the users for the team page
```
Comment (Nicofuma) [inline phpBB/memberlist.php]: this event doesn't allow you to edit any SQL query

Comment (Senky) [inline phpBB/memberlist.php]: In fact, it allows. Just see what `$use_contact_fields` variable modifies - parameter of `$cp->generate_profile_fields_template_data` call, which modifies SQL query inside that method.

Comment (Nicofuma) [inline phpBB/memberlist.php]: yes but it doesn't allow you to edit the sql query itself but if yes or no the contact fields should be used

Comment (Senky) [inline phpBB/memberlist.php]: True. So the problem is just the description of the event? I can change that easily.

Comment (Nicofuma) [inline phpBB/memberlist.php]: yep

**Pull request merged**